sdk/ml/azure-ai-ml/azure/ai/ml/_schema/workspace/networking.py

```diff
@@ -114,6 +114,63 @@ def service_tag_dest2dict(self, service_tag, protocol, port_ranges):
         return service_tag_dest
 
 
+@experimental
+class PrivateEndpointDestinationSchema(metaclass=PatchedSchemaMeta):
+    service_resource_id = fields.Str(required=True)
+    subresource_target = fields.Str(required=True)
+    spark_enabled = fields.Bool(required=True)
+
+
+@experimental
+class PrivateEndpointOutboundRuleSchema(metaclass=PatchedSchemaMeta):
+    name = fields.Str(required=True)
+    type = StringTransformedEnum(
+        allowed_values=[
+            OutboundRuleType.PRIVATE_ENDPOINT,  # use allowed values to differentiate for unionfield ordering
+        ],
+        casing_transform=camel_to_snake,
+        metadata={"description": "outbound rule type."},
+    )
```

Comment:
Shouldn't type be a constant for this class?

Reply:
applied, thanks. was using enum with only one option to properly match type used by UnionField. Tested with marshmallow constant field and that can work as long as order for unionfields is correct. applied